api-specificatie/openapi.yaml

```diff
@@ -267,35 +267,30 @@ paths:
 
-  /panden:
-    post:
-      requestBody:
-        description: Geometrisch punt waarop de panden bepaald moeten worden.
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/geometrie/geometrie.yaml#/components/schemas/Point'
+  /panden/{pandIdentificatie}:    
+    get:
       security:
         - apiKeyBAG: []
+      operationId: raadpleegPand
       tags:
-        - Pand
-      summary: bevragen van één of meer panden, op basis van de geometrische locatie.
-      operationId: raadpleegPandOpLocatie
-      description: "Raadpleeg een pand op basis van een geometrische locatie (punt).
+        - Panden
+      summary: Raadpleeg Pand o.b.v. een identificatie
+      description: "Raadpleeg een pand op basis van de identificatie
         Geleverd worden de actuele gegevens van één of meer panden evt. met gerelateerde verblijfsobject en adres. 
+        Parameter pandIdentificatie bevat de 16 cijferige identificatie van een pand.</li>
         Als expand=pand dan worden de gerelateerde pand objecten als geneste resources geleverd.<br/>
         Als expand=adres dan wordt het gerelateerde adres als geneste resources geleverd.<br/>
         Als expand=true dan worden het gerelateerde adres en pand(en) objecten als geneste resources geleverd.<br/>
         De contentCrs in de header moet zijn epsg:28992"
       parameters:
-        - $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/parameters.yaml#/expand'
+        - $ref: '#/components/parameters/pandIdentificatie'
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/api-version"
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/expand"
+        - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields"
         - $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/headers.yaml#/contentCrs'
-
-      # Antwoord pandGeometrie
       responses:
         '200':
-          description: Bevraging pandGeometrie geslaagd
+          description: Geslaagd
           headers:
             api-version:
               $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/headers.yaml#/api_version'
```

Comment:
required: true in de definitie van api_version zorgt voor code generatie fout in NSwagStudio
idem voor required: false in de definitie van warning

Reply:
Required: true is volgens de OAS v3 specificatie gewoon toegestaan in response headers. Het lijkt erop also de NSwagStudio tool de specificatie niet goed interpreteert. Het kan ook zijn dat evt. configuratie van de tool niet correct is. 
Het lijkt mij in dit geval niet de bedoeling om de specificatie aan te passen, zodanig dat de tooling werkt, maar dat er andere tooling wordt gebruikt of (indien mogelijk) de configuratie van de tooling wordt aangepast.
Zie:
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#headerObject
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#parameterObject